app/views/categories/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row">
+    <% @categories.each do |category| %>
+      <div class="col-md-12 mb-3">
+        <div class="card w-100">
+          <div class="card-body d-flex">
+            <div class="flex-grow-1 name">
+              <img src="<%= category.icon %>"class="card-img-top responsive-img" alt="icon">
+              <%= link_to category.name, category_transaction_items_path(category.id), method: :get, class:"card-link text-decoration-none" %>
+            </div>
+            <div class="text-right amount">
+              <%= category.total_amount %>
+            </div>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
+<div class="btn-container">
+    <%= link_to new_category_path, class:"link-container" do %>
+        <button class="btn btn-success">Add Category</button>
+    <% end %>
+</div>
```

Comment:
- [ ] Kindly position this button to the bottom section of the screen so that your design can match the provided Behance design.

![image](https://user-images.githubusercontent.com/86318284/219852363-a3b46bc8-db21-41cd-b5c0-9e7df771b9e9.png)

![image](https://user-images.githubusercontent.com/86318284/219852447-44e892b8-777d-48ec-a986-2679399d9b5c.png)